submission-panels.md

```diff
@@ -0,0 +1,80 @@
+# Submission-panels Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/configuration/submission-definitions**   
+
+Provide access to the configured submission definitions. It returns the list of existent submission-definitions.
+
+Example: to be provided
+
+## Single Submission-panels 
+**/api/configuration/submission-definitions/<:definition-name>**
+
+Provide detailed information about a specific submission-definition. The JSON response document is as follow
+```json
+{
+  "name": "traditional",
+  "panels": [
+  {
+  	id: "id-of-the-input-form-page",
+  	header: "First page",
+  	mandatory: true,
+  	type: "inputform",
+  	scope: null,
+  	_links: {
+  		"configuration" : "<dspace-url>/configuration/inputforms/<:id-of-the-input-form-page>" 
+  	}
+  },
+  {
+    id: "id-of-the-input-form-page",
+  	header: "Second page",
+  	mandatory: true,
+  	type: "inputform",
+  	scope: null,
+  	_links: {
+  		"configuration" : "<dspace-url>/configuration/inputforms/<:id-of-the-input-form-page>" 
+  	}
+  },
+  {
+    id: "id-of-the-upload-panel",
+  	header: "Files and access condition",
+  	mandatory: true,
+  	type: "uploadWithEmbargo",
```

Comment:
It would be nice if we could also expose a `configuration` link for this panel type. For now, we can just return a fixed response that the UI can use to render that panel.

Reply:
Not sure if this is as easily achieved as stated.  We don't really have defined "configuration" for an upload field, or embargo settings, CC license field, *yet*.  

Longer term, though, it would be nice to think about (as it could let us drop the idea of a page/panel `type` and just configure each panel as a set of fields (and one field could be a "file upload" field).  That would be a powerful idea, cause it'd also let you simplify the deposit into a single *page/panel* (which could include metadata entry fields, upload fields, embargo field... all intermixed together in whatever order you wanted)

Reply:
The configuration link was one of the suggestion accepted in the latest development see
https://github.com/DSpace/Rest7Contract/pull/11/commits/3fd9c24e91de383b514de0646e3f522c562e5284
https://github.com/DSpace/Rest7Contract/pull/11/files#diff-71070ef064894a46feca56c175a2464fR26

The configuration for the upload panel, the license, etc. still need to be analyzed (there are limited configuration options in the dspace.cfg file for that) but it can be eventually exposed using the same link (this was the idea).